app/services/people/finder.rb

```diff
@@ -19,14 +19,27 @@ def apply_filters(scope)
     end
 
     def apply_police_national_computer_filters(scope)
-      return unless filter_params.key?(:police_national_computer)
+      return unless filter_params.key?(:police_national_computer) || filter_params.key?(:nomis_offender_no)
```

Comment:
This return is now superfluous as the body checks both values in if statements

Reply:
If I remove this, the behavior would be: return all the profile if there are not filters.   
That actually it makes sense.
I just need to check if there are not any calls that relies on the current behavior: return nil if there are not filters.

Reply:
removed! thans